spec/migrations/240814115647-disable_reverse_proxy_by_default_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+migration '240814115647-disable-reverse-proxy-by-default' do
+  scenarios %w[foreman-proxy-content] do
+    context 'reset reverse_proxy to default value to disable by default' do
```

Comment:
`context` is used to describe the context, typically the `let` blocks. How about
```suggestion
    context 'with a stored answer' do
```

Not asking for it here, but from that you can also easily see the context `without a stored answer`. That could have an `it "doesn't crash" do` or something like that.

At least, that's how I interpret them and structure my RSpec tests. See https://rspec.help/getting-to-know-describe-context/ or https://rspec.help/rspec/basics/ if you're interested in more.